Makefile

```diff
@@ -439,7 +439,7 @@ FYPPFLAGS ?= -n
 	$(CXX) -c $(CXXFLAGS) $<
 
 libcusmm.o: libcusmm.cpp parameters.h cusmm_kernels.h
-	$(CXX) -c $(CXXFLAGS) -DARCH_NUMBER=$(ARCH_NUMBER) $<
+	$(CXX) -c $(CXXFLAGS) -fopenmp -DARCH_NUMBER=$(ARCH_NUMBER) $<
```

Comment:
Please note that this most likely breaks compilation with PGI compilers (which we do not support yet since they are unable to compile DBCSR, but their devs are working on it).
Plus we might also have to add a check to the `CMakeLists.txt` that OMP is required when building with CUDA support.

Reply:
Right, good point ... thanks for that! 
- Keeping the option of building **with** CUDA support but **without** OMP support is probably desirable, I'll make this possible using appropriate `#ifdef`s in `libcusmm.cpp`. 
- To fix this foreseeable PGI issue, we should probably let the OMP flags be defined appropriately in the `Makefile.inc` rather than hard-coded in the `Makefile`. I'll fix this.